# Serie 3

## Aufgabe 3.1
Die monatlichen Aufwendungen X [CHF] für den Wasserverbrauch eines 2-Personenhaushalts seien durch eine Zufallsvariable mit der folgenden Wahrscheinlichkeitsdichtefunktion gegeben:

$$f(x)=\left\{
    \begin{matrix}
    cx(15-\frac{x}{4})\ && \text{falls}\ 0\leq x\leq 60 \\
    0 && \text{sonst}
    \end{matrix}
\right. $$

### a)

Welchen Wert muss $c$ annehmen?

$$\int\limits_0^{60}{cx(15-\frac{x}{4})} = 1 $$

$$\left.\frac{15\cdot c\cdot x^2}{2}-\frac{c\cdot x^3}{12}\right|_0^{60} = 1 $$

$$\frac{15\cdot c\cdot 60^2}{2}-\frac{c\cdot 60^3}{12} = 1 $$

$$\frac{54000\cdot c}{2} - 18000\cdot c = 1 $$

$$27000\cdot c - 18000\cdot c = 1$$

$$c = \frac{1}{9000}$$

In [2]:
print(str(15*60**2))
print(str(60**3/12))

54000
18000.0


### b)

Geben Sie die Verteilungsfunktion $F$ der Zufallsvariablen $X$ an.

$$ F(x)=\left\{
    \begin{matrix}
    0 && \text{falls}\ x<0\\
    \frac{1}{9000}(\frac{15}{2}x^2-\frac{1}{12}x^3)\ && \text{falls}\ 0\leq x\leq 60\\
    1 && \text{falls}\ x>60
    \end{matrix}
\right. $$

### c)

Welcher Wert der Aufwendungen wird nur mit 10% Wahrscheinlichkeit überschritten?

Gesucht ist ein $a$ sodass $P(X>a)=0.1$ also ist $P(X\leq a)=0.9$, somit muss gelten $F(a)=0.9$. Mit der Formel aus b):

$$\frac{1}{9000}(\frac{15}{2}a^2-\frac{1}{12}a^3) = 0.9$$

$$\frac{15}{2}a^2-\frac{1}{12}a^3 = 0.9\cdot 9000$$

$$\frac{15}{2}a^2-\frac{1}{12}a^3 = 8100$$

$$\frac{15}{2}a^2-\frac{1}{12}a^3 - 8100 = 0$$

In [3]:
import sympy as sym

a = sym.Symbol('a')
sym.solveset(15/2*a**2-1/12*a**3-8100)

{-28.6249720273366, 48.2519936604545, 70.3729783668821}

Davon liegt nur $a = 48.252$ im Wertebereich von $F(x)$ und ist somit unsere gesuchte Grenze.

### d)

Wie hoch sind die erwarteten monatlichen Aufwendungen für den Wasserverbrauch eines 2-Personenhaushalts?

Der Erwartungswert für stetige Zufallsvariablen $X$ ist wie folgt definiert:

$$\int\limits_{-\infty}^{\infty}{x \cdot f(x)\ dx} $$

Da unsere Funktion $f(x)$ aber nur im Intervall $[0,60]$ definiert ist, erhalten wir folgendes Integral:

$$\int\limits_0^{60}{x\cdot \frac{1}{9000}x\ (15-\frac{x}{4})}$$

$$= \frac{1}{9000}\int\limits_0^{60}{x^2\ (15-\frac{x}{4})}$$

$$= \frac{1}{9000}(\left. 5x^3-\frac{1}{16}x^4\right|_0^{60})$$

$$= 30$$

In [4]:
5*60**3

1080000

In [5]:
1/16*60**4

810000.0

In [6]:
1/9000*(1080000-810000)

30.0

## Aufgabe 3.2

In derStadt Luzern gibt es bekanntlich viele Baustellen. Die Dauer $X$ der Arbeiten bei einer Baustelle liege zwischen 0 und 20 Wochen. Die Dichte $f(x)$ habe die folgende Form:

![Dauer Arbeiten auf Baustellen in Luzern](img/ArbeitenBaustelleLuzern.png)

### a)
Begründen Sie, warum $c = 0.1$ ist und schreiben Sie die Dichte $f(x)$ explizit auf.

Da die Fläche unter der Wahrscheinlichkeitsdichtefunktion 1 ergeben muss.

Wenn wir das Ganze als Dreiecksfläche anschauen erhalten wir $\frac{c\cdot20}{2}=1$. und somit $c=\frac{1}{10}$. Die Wahrscheinlichkeitsdichtefunktion ist also:

$$f(x)=\left\{
    \begin{matrix}
    0\ && x<0\ \text{und}\ x>20 \\
    -\frac{1}{200}x+\frac{1}{10} && 0\leq x\leq 20
    \end{matrix}
\right. $$

### b)
Berechnen Sie die Wahrscheinlichkeit, dass die Bauzeit $X$ weniger als
1. 5 Wochen
2. 10 Wochen

beträgt.

$$g(t) = F(t) = \int\limits_{0}^{t}{-\frac{1}{200}x+\frac{1}{10}\ dx} = -\frac{1}{400}t^2+\frac{1}{10}t $$

Und somit:

$$g(5) = -\frac{5^2}{400}+\frac{5}{10}$$

$$g(10) = -\frac{10^2}{400}+\frac{10}{10}$$

In [7]:
-(5**2)/400+5/10

0.4375

In [8]:
-10**2/400+1

0.75

### c)

Skizzieren Sie die kumulative Verteilungsfunktion.

In [9]:
import numpy as np
import matplotlib.pyplot as plt

# Sample t between 0 and 20
t = np.arange(0., 20.0, 0.5)

plt.plot(t, -t**2/400+t/10, linewidth=1.0)
plt.show()

<Figure size 640x480 with 1 Axes>

### d)

Berechnen Sie den Erwartungswert, den Median und die Standardabweichung der Dauer $X$.

Wenn X stetig ist, ist der Erwartungswert $E(X)$ folgendermassen definiert:

$$E(X) = \int_{-\infty}^{\infty}xf(x)\text{d}x $$

Auf unser Problem angewandt bedeutet dies:

$$E(X) = \int_0^{20}{-\frac{1}{200}x^2+\frac{1}{10}x\ \text{d}x} = \frac{20}{3} \approx 6.6666$$

In [10]:
import scipy.integrate as integrate
integrate.quad(lambda x:-1/200*x**2+1/10*x, 0, 20)

(6.666666666666666, 7.401486830834377e-14)

Die Varianz für $E(X)$ ist definiert als $Var(X) = E(X^2)-(E(X)^2$

$E(X)$ haben wir im ersten Schritt schon berechnet.

$$E(X^2) = \int_0^{20}{x^2 f(x) \text{d}x} = \int_0^{20}{-\frac{1}{200}x^3+\frac{1}{10}x^2\text{d}x} = \frac{200}{3}$$

$$ Var(X) = E(X^2)-(E(X)^2 = \frac{200}{3}-(\frac{20}{3})^2 = \frac{200}{9} \approx 22.2222$$

In [11]:
200/3-(20/3)**2

22.22222222222222

Standardabweichung $\sigma(X) =\sqrt{Var(X)}=\sqrt{200/9}=\sqrt{2}\cdot\frac{10}{3}\approx4.7140$

### e)

$K = 40′000\cdot\sqrt{X}$ entspreche dem Betrag in Franken, den die Arbeiten bei einer Baustelle kosten. Wie gross ist die Wahrscheinlichkeit, dass die Arbeiten bei einer Baustelle höchstens 120’000.- Fr. kosten?

$$\int\limits_{0}^{t}{-\frac{1}{200}x+\frac{1}{10}\ dx}$$

In [12]:
print((120000/40000)**2)
integrate.quad(lambda x:-1/200*x+1/10, 0, 9)[0]

9.0


0.6975000000000001

Die vorgeschlagene Verteilung ist nur ein Modell. Man könnte die Dauer der Bauarbeiten zum Beispiel auch als exponentialverteilt annehmen.

### f)
 Für welchen Parameter $\lambda$ hat die Exponentialverteilung denselben Erwartungswert wie die bisherige angenommene Verteilung?

$$E(X) = 6.666$$
Erwartungswert einer Exponentialverteilung: $ E(X) = \frac{1}{\lambda}$
$$\frac{1}{\lambda} = 6.666$$
$$\lambda = 0.15$$

In [13]:
1/6.666666666666666

0.15000000000000002

 ### g)
Berechnen Sie mit der gefundenen Exponentialverteilung nochmals Teilaufgabe e).

In [14]:
integrate.quad(lambda x:0.15*np.exp(-0.15*x), 0, 9)[0]

0.7407597393541085

## Aufgabe 3.3

Eine Maschine hat eine exponentialverteilte Lebensdauer $T_1$ (in Stunden) mit Parameter $\lambda = 0.04$.

### a)
Berechnen Sie den Median und den Erwartungswert. Mit welcher Wahrscheinlichkeit überlebt das System seine Lebenserwartung?

Median:
$$ 50\% =\int\limits_{0}^{m}{\lambda\cdot e^{-\lambda x}\ dx}$$
$$ 0.5 =\int\limits_{0}^{m}{0.04\cdot e^{-0.04 x}\ dx}$$
$$ 0.5 = e^{0} - e^{-0.04*m} $$
$$ m = 17.329 $$

Erwartungswert:
$$ E[X] = \frac{1}{\lambda}$$
$$ E[X] = 25 $$

Mit welcher Wahrscheinlichkeit liegt die Lebensdauer des Systems im Bereich
$\text{E}[T_1] \pm \sigma T_1$ ?

$$ P(T_1 > E(T_1)) = P(T_1 > \frac{1}{\lambda_1})$$
$$ = 1 - P(T_1 \leq \frac{1}{\lambda_1})$$
$$ = 1 - kumulativeVerteilungsfunktion(\frac{1}{\lambda_1})$$
$$ = 1 - (1 - e^{-\lambda\cdot \frac{1}{\lambda}})$$
$$ = e^-1 \approx 0.368$$

In [15]:
1/np.exp(1)

0.36787944117144233

### c)
Nun betrachten wir eine zweite Maschine mit der Lebensdauer $T_2$ (in Stunden) mit der folgenden Dichte:

$$f_{T_2}(t)=\left\{
    \begin{matrix}
    0\ && t \leq 0 \\
    c_2\cdot e^{-\frac{1}{1000}t} && t \geq 0
    \end{matrix}
\right. $$

Wie gross muss $c_2$ sein? Berechnen Sie den Erwartungswert und die Standardabweichung der Zufallsvariablen $T_2$.

$$\int\limits_{0}^{\infty}f_{T_2}(t) dt = 1 \rightarrow c_2\cdot 1000=1$$
$$ c_2 = \frac{1}{1000} $$

Erwartungswert: $E(T_2) = \frac{1}{c_2} = 1000$

Varianz $\sigma_{T_2}^2 = \frac{1}{c_2^2} \rightarrow \sigma = 1000$

### d)
Wir betrachten nun das System bestehend aus den beiden Maschinen mit Lebensdauern $T_1$ und $T_2$, welche unabhängig voneinander funktionieren. Berechnen Sie die Wahrscheinlichkeit, dass nach 200 Stunden beide Maschinen noch funktionieren.

Da Sie unabhängig sind: 
$$P(T_1 \geq 200, T_2 \geq 200) = P(T_1 \geq 200)\cdot P( T_2 \geq 200)$$
$$ (1 - \int\limits_{0}^{200}f_{T_1}(t) dt)(1 - \int\limits_{0}^{200}f_{T_2}(t) dt) $$
$$ (1 - (1-e^{-200\lambda_1}))(1 - (1-e^{-200\lambda_2})$$
$$ e^{-200\lambda_1}\cdot e^{-200\lambda_2}$$

In [16]:
import numpy as np
np.exp(-200*1/25)*np.exp(-200*1/1000)

0.0002746535699721423

## Aufgabe 3.4

Aufgrund langjähriger Untersuchungen ist bekannt, dass der Bleigehalt $X$ in einer Bodenprobe annähernd normalverteilt ist. Ausserdem weiss man, dass der Erwartungswert 32ppb beträgt und dass die Standardabweichung 6ppb beträgt.

### a)
Machen Sie eine Skizze der Dichte von X, und zeichnen Sie die Wahrscheinlichkeit, dass eine Bodenprobe zwischen 26 und 38 ppb Bleienthält, in die Skizze ein.

## b)
Wie gross ist die Wahrscheinlichkeit, dass eine Bodenprobe höchstens 40ppb
Schwermetall enthält? _Hinweis_: Benützen Sie die **Python**-Funktion

```python
norm.cdf()
```

ÜberprüfenSiedasResultat, indemSiediestandardisierte Zufallsvariable Z ein-
führen und mit der Standardnormalverteilung die Wahrscheinlichkeit berech-
nen.

### c)

Wie gross ist die Wahrscheinlichkeit, dass eine Bodenprobe höchstens 27 ppb
Schwermetall enthält?

### d)
Welcher Bleigehalt wird mit einer Wahrscheinlichkeit von 97.5% unterschritten?
Das heisst, bestimmen Sie dasjenige c, so dass die Wahrscheinlichkeit, dass der Bleigehalt kleiner oder gleich c ist, genau 97.5% beträgt. _Hinweis_: Benützen Sie die **Python**-Funktion
```python
norm.ppf()
```

### e)
Welcher Bleigehalt wird mit einer Wahrscheinlichkeit von 10% unterschritten?

### f)
Wie gross ist die Wahrscheinlichkeit, die in Aufgabe a) eingezeichnet wurde?

## Aufgabe 3.5

Wir gehen davon aus, dass bei der Detektion eines digitalen Signals das Hintergrundrauschen einer Normalverteilung folgt, und zwar mit einem Mittelwert von 0 Volt und einer Standardabweichung von 0.45 Volt. Das System geht davon aus, dass eine digitale 1 übertragen worden ist, wenn die Spannung 0.9 Volt überstiegen worden ist.